virtcontainers/mount.go

```diff
@@ -442,3 +443,16 @@ func isEmptyDir(path string) bool {
 	}
 	return false
 }
+
+func IsMountedNoExec(path string) (bool, error) {
```

Comment:
I think that there is no need to export (make public) this function, right?

Reply:
yep. you're right. ;). update asap.